2022-01-14 13:41:12,321 INFO    MainThread:414867 [wandb_setup.py:_flush():71] setting env: {}
2022-01-14 13:41:12,322 INFO    MainThread:414867 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 13:41:12,322 INFO    MainThread:414867 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 13:41:12,322 INFO    MainThread:414867 [wandb_init.py:_log_setup():371] Logging user logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_134112-3ohammdd/logs/debug.log
2022-01-14 13:41:12,322 INFO    MainThread:414867 [wandb_init.py:_log_setup():372] Logging internal logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_134112-3ohammdd/logs/debug-internal.log
2022-01-14 13:41:12,323 INFO    MainThread:414867 [wandb_init.py:init():404] calling init triggers
2022-01-14 13:41:12,323 INFO    MainThread:414867 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-14 13:41:12,323 INFO    MainThread:414867 [wandb_init.py:init():460] starting backend
2022-01-14 13:41:12,323 INFO    MainThread:414867 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-14 13:41:12,329 INFO    MainThread:414867 [backend.py:ensure_launched():216] starting backend process...
2022-01-14 13:41:12,332 INFO    MainThread:414867 [backend.py:ensure_launched():221] started backend process with pid: 414928
2022-01-14 13:41:12,333 INFO    MainThread:414867 [wandb_init.py:init():469] backend started and connected
2022-01-14 13:41:12,335 INFO    MainThread:414867 [wandb_init.py:init():533] updated telemetry
2022-01-14 13:41:12,344 INFO    MainThread:414867 [wandb_init.py:init():563] communicating current version
2022-01-14 13:41:12,742 INFO    MainThread:414867 [wandb_init.py:init():568] got version response 
2022-01-14 13:41:12,742 INFO    MainThread:414867 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-14 13:41:13,025 INFO    MainThread:414867 [wandb_init.py:init():606] starting run threads in backend
2022-01-14 13:41:13,079 INFO    MainThread:414867 [wandb_run.py:_console_start():1810] atexit reg
2022-01-14 13:41:13,079 INFO    MainThread:414867 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-14 13:41:13,079 INFO    MainThread:414867 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-14 13:41:13,080 INFO    MainThread:414867 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-14 13:41:13,080 INFO    MainThread:414867 [wandb_init.py:init():633] run started, returning control to user process
2022-01-14 13:41:13,111 INFO    MainThread:414867 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-14 13:41:13,112 INFO    MainThread:414867 [wandb_run.py:_restore():1752] restore
2022-01-14 13:41:15,587 INFO    MainThread:414867 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 13:41:15,769 INFO    MainThread:414867 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 13:41:16,033 INFO    MainThread:414867 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 3910
}

2022-01-14 13:41:16,135 INFO    MainThread:414867 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 3910
}

2022-01-14 13:41:16,236 INFO    MainThread:414867 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3910
  total_bytes: 3910
}

2022-01-14 13:41:16,339 INFO    MainThread:414867 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3910
  total_bytes: 3910
}

2022-01-14 13:41:16,443 INFO    MainThread:414867 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3910
  total_bytes: 3910
}

2022-01-14 13:41:16,545 INFO    MainThread:414867 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3910
  total_bytes: 3910
}

2022-01-14 13:41:16,648 INFO    MainThread:414867 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3910
  total_bytes: 3910
}

2022-01-14 13:41:17,108 INFO    MainThread:414867 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3910
  total_bytes: 3910
}

2022-01-14 13:41:17,496 INFO    MainThread:414867 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3910
  total_bytes: 3910
}
local_info {
}

2022-01-14 13:41:18,547 INFO    MainThread:414867 [wandb_run.py:_append_files():2180] logging synced files
